metadata.yaml

```diff
@@ -22,11 +22,17 @@ containers:
     mounts:
       - storage: data
         location: /data
+  synapse-nginx:
+    resource: synapse-nginx-image
 
 resources:
   synapse-image:
     type: oci-image
     description: OCI image for Synapse
+  synapse-nginx-image:
+    type: oci-image
+    description: Docker image for nginx Synapse
```

Comment:
Can we say "OCI image" rather than "Docker image" here?

Reply:
Sure!